packages/formation/js/sidenav.js

```diff
@@ -97,15 +89,10 @@ class SideBarMenu {
     this.loadKeyNodes(trigger);
     this.menu.classList.add('va-sidebarnav--opened');
     this.menu.setAttribute('aria-hidden', 'false');
-    document
-      .getElementsByClassName('va-btn-sidebarnav-trigger')[0]
-      .setAttribute('hidden', 'true');
     document.getElementsByTagName('body')[0].style.overflow = 'hidden';
     this.addCloseMenuListener();
-    if (trigger.className.includes('va-btn-sidebarnav-trigger')) {
```

Comment:
I see no reason to conditionally move focus to the menu's "close" button as soon as it opens.